### ISSUE TITLE
[Suggestion] Make Entangleaporter channels harder to delete. (I hate myself now)

### ISSUE THREAD

**Issue by TheUnderTaker11** [mekanism-mod/Mekanism#5294]
#### Issue description:
Make it harder to delete entangleaporter channels. I was going about my daily business, placing an entangleaporter for my Quarry, and suddenly I move my mouse slightly to far to the right and all the sudden my energy channel got deleted. Now that means I have to try and track down EVERY SINGLE ENTANGLEAPORTER I've placed since I started using them, and manually set the channel again.      

It's probably something you haven't realized could be a problem but it is far to easy to completely screw yourself over with a slight misclick.

#### Steps to reproduce:
1. Place around 40-50 quantum entangleaporters over many different dimensions that all use an energy channel.
2. Place another one for whatever you are currently trying to power, but move your mouse ever so slightly to far to the right when setting the channel.
3. Realize you have now just completely nuked your power delivery system.

#### Version (make sure you are on the latest version before reporting):
**Forge:** 1.12.2
**Mekanism:** 9.4.13.349

Basically please add a 2 click process or something to delete channels, it just absolutely screwed me that I accidentally deleted my power channel that is hooked up to a bare minimum of 40 Entangleaporters.       
And in case this comes off as rude I don't blame the mod author or anything, it would just be a HUGE help to stop anyone from having to go through what I just did.
